uber/models.py

```diff
@@ -401,6 +401,11 @@ def apply(self, params, *, bools=(), checkgroups=(), restricted=True, ignore_csr
                             value = None
                         else:
                             value = float(value)
+                    elif isinstance(column.type, Numeric):
+                        if value == '':
+                            value = None
+                        elif value.endswith('.0'):
+                            value = int(float(value))
```

Comment:
INNOCENT WHISTLING

This was one of those changes that had to be done in a hurry so wasn't super well thought through. Numeric is a weird column type and I would like to figure out an alternate solution after pre-reg launches.